zerver/logging_handlers.py

```diff
@@ -51,7 +51,7 @@ def emit(self, record):
                 method = request.method,
                 path = request.path,
                 data = request.GET if request.method == 'GET'
-                                   else filter.get_post_parameters(request),
+                else filter.get_post_parameters(request),
```

Comment:
Using parentheses would give you the indentation back, which would be slightly clearer.